lib/segment/src/fixtures/payload_fixtures.rs

```diff
@@ -121,6 +121,18 @@ pub fn random_vector<R: Rng + ?Sized>(rnd_gen: &mut R, size: usize) -> DenseVect
     (0..size).map(|_| rnd_gen.gen()).collect()
 }
 
+pub fn random_multi_vector<R: Rng + ?Sized>(
+    rnd_gen: &mut R,
+    vector_size: usize,
+    num_vector_per_points: usize,
```

Comment:
I propose to use `1..num_vector_per_points` range and generate random mutlivector with random amount of vectors

Reply:
There is an invariant that all vectors within a point have the same dimensions.

Do you want the test using this to have points with different number of vectors maybe?

Reply:
Done this approach https://github.com/qdrant/qdrant/pull/3973/commits/754d10287a24a85c15d75cffb5e4244b610c078d